File: Problemas/486A - Calculating Function/486A_Calculating_Function.py
# 486A - Calculating Function
# https://codeforces.com/problemset/problem/486/A

# Inputs:
# 1) Número (N)
n = int(input())

# Si N es par, entonces la respuesta es N/2
if(n % 2 == 0):
    print(n // 2)

# Si N es impar, entonces la respuesta es -((N+1)/2)
else:
    print(-((n+1)//2))

File: Problemas/486A - Calculating Function/486A_Calculating_Function.cpp
// 486A - Calculating Function
// https://codeforces.com/problemset/problem/486/A

#include <stdio.h>
#include <stdlib.h>
#include <bits/stdc++.h>
#include <cmath>
using namespace std;

int main(){

    // Input:
    // 1) N = Número
    int n;
    cin >> n;

    // Si N es par, entonces la respuesta es N/2
    // Si N es impar, entonces la respuesta es -((N+1)/2)
    if(n % 2 == 0) {
        cout << n/2 << endl;
    } else {
        cout << -((n+1)/2) << endl;
    }

    return 0;
}
